src/layouts/city.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * This is an example of how to create a template that makes use of streams data.
+ * The stream data originates from Yext's Knowledge Graph. When a template in
+ * concert with a stream is built by the Yext Sites system, a static html page
+ * is generated for every corresponding (based on the filter) stream document.
+ *
+ * Another way to think about it is that a page will be generated using this
+ * template for every eligible entity in your Knowledge Graph.
+ */
+
+import type {
+  Template,
+  GetPath,
+  TemplateConfig,
+  TransformProps,
+  GetHeadConfig,
+  Stream,
+} from "@yext/pages";
+import "src/index.css";
+import { defaultHeadConfig } from "src/common/head";
+import type {
+  DirectoryProfile,
+  LocationProfile,
+  TemplateProps,
+  TemplateRenderProps,
+} from "src/types/entities";
+import { Main } from "src/layouts/main";
+import DirectoryLayout from "src/layouts/directory";
+
+/**
+ * Required when Knowledge Graph data is used for a template.
+ */
+export const configBuilder: (
```

Comment:
I've also been splitting this out into it's own file, since the list of fields can get rather long & make this file a bit unwieldy

Reply:
I don't disagree but I'm going to leave as is for now